File: TechModule/SoftwareTechnologies/Exam/JS/models/Trip.js
const mongoose = require('mongoose');

let tripSchema = mongoose.Schema({
    origin: {type: "string", required:true, minlength: 1},
    destination: {type: "string", required:true, minlength: 1},
    business: {type: "Number", required:true},
    economy: {type: "Number", required:true},
});

let Trip = mongoose.model('Trip', tripSchema);

module.exports = Trip;

File: TechModule/SoftwareTechnologies/Exam/C#/TrainSystem/Models/Trip.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace TrainSystem.Models
{
    public class Trip
    {

        [Key]
        public int Id { get; set; }

        [Microsoft.Build.Framework.Required]
        [MinLenght(1)]
        public string Origin { get; set; }
        [Microsoft.Build.Framework.Required]
        [MinLenght(1)]
        public string Destination { get; set; }

        public int Business { get; set; }
        public int Economy { get; set; }

    }

    public class MinLenghtAttribute : Attribute
    {
        public MinLenghtAttribute(int i)
        {
            throw new NotImplementedException();
        }
    }
}